components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1565,6 +1565,7 @@ real(rtype) function polysvp1(T,i_type)
 
        print*
        print*,'** polysvp1 i_type must be 0 or 1 but is: ',i_type,T
+       print*,' T = ',T
```

Comment:
We can probably get rid of the "T" in the line above so we don't print the temp twice.  Or better yet, combine these into one print statement so that they don't get separated when multiple cores issue the same print statement at the same time.

Reply:
Probably my fault as well.  How about:
       print*,'** polysvp1 i_type must be 0 or 1 but is %d: ',i_type
I had added the T when I was searching for a bug, but it isn't generally relevant.

Reply:
Ok, fixed this.